### PR TITLE
ZK-3814: Java 8 default interface methods not recognized as managed b…

### PR DESCRIPTION
**Pull request by scribetw**
…ean properties in EL

### CODE REVIEW

File: zel/src/org/zkoss/zel/Util.java [hunk 1]
```diff
@@ -784,4 +787,10 @@ public int compareTo(MatchResult o) {
             return cmp;
         }
     }
+
+    // A workaround for https://bugs.openjdk.java.net/browse/JDK-8071693
+    // ZK-3814: Java 8 default interface methods not recognized as managed bean properties in EL
+    static PropertyDescriptor[] getPropertyDescriptors(Class<?> beanClass) throws IntrospectionException {
```
Comment (klyvechen) [inline zel/src/org/zkoss/zel/Util.java]: A workaround??

Comment (scribetw) [reply zel/src/org/zkoss/zel/Util.java]: Yes, a JDK bug

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**